modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase

```diff
@@ -1758,6 +1758,60 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-11961. User can search products on the Grouped tab of grouped products"
+	@priority = 5
+	test CanSearchProductsOnGroupedProducts {
+		property portal.acceptance = "true";
+
+		task ("Given a Minium site is created") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+		}
+
+		task ("And a grouped type of product is created consisting of three products") {
+			CommerceJSONProductsAPI._addCommerceProduct(
+				catalogName = "Minium",
+				productDescription = "Full Description",
+				productName = "Grouped Product",
+				productType = "Grouped",
+				shortDescription = "Short Description",
+				sku = "GROUPEDSKU");
+
+			CommerceProducts.openProductsAdmin();
+
+			CommerceNavigator.searchEntry(entryName = "Grouped");
```

Comment:
Hi @baharturk, we cannot use the macro, because the full product name cannot be searched on the page, see:
[doubt.webm](https://github.com/liferay-commerce/liferay-portal/assets/85482945/dd080e38-29f6-461e-8c83-04d7620b4bd1)

Reply:
ok, got it, that's fine.

Reply:
Thank you!!!